source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -115,15 +129,19 @@ Http::FilterDataStatus Filter::onData(ProcessorState& state, ProcessingMode::Bod
       }
 
       // The body has been buffered and we need to send the buffer
+      ENVOY_LOG(debug, "Sending request body message");
       state.addBufferedData(data);
       state.setCallbackState(ProcessorState::CallbackState::BufferedBody);
       state.startMessageTimer(std::bind(&Filter::onMessageTimeout, this),
                               config_->messageTimeout());
       sendBodyChunk(state, *state.bufferedData(), true);
-    } else {
-      ENVOY_LOG(trace, "onData: Buffering");
+      // Since we just manually adjusted the buffer, don't buffer again
```

Comment:
Maybe `Since we already moved the buffer into ProcessorState...`?

also nit: end with period

Reply:
Clarified comments.